packages/core/admin/admin/src/content-manager/components/NonRepeatableComponent/index.js

```diff
@@ -20,7 +20,7 @@ const NonRepeatableComponent = ({ componentUid, isFromDynamicZone, isNested, nam
 
   return (
     <Box
-      background={isFromDynamicZone ? 'neutral0' : 'neutral100'}
+      background={isFromDynamicZone ? '' : 'neutral100'}
```

Comment:
Component in dynamic zone is in charge of the background to allow easier management of the bottom border radius needed